hypervisor/arch/x86/io.c

```diff
@@ -387,11 +416,26 @@ static void empty_io_handler_list(struct vm *vm)
 	vm->arch_vm.io_handler = NULL;
 }
 
+/**
+ * @brief Free I/O bitmaps and port I/O handlers of \p vm
+ *
+ * @param vm The VM whose I/O bitmaps and handlers are to be freed
+ */
 void free_io_emulation_resource(struct vm *vm)
 {
 	empty_io_handler_list(vm);
 }
 
+/**
+ * @brief Allow a VM to access a port I/O range
+ *
+ * This API enables direct access from the given \p vm to the port I/O space
+ * starting from \p address_arg to \p address_arg + \p nbytes - 1.
```

Comment:
Should we use `port_address` here instead of `address_arg`?

Reply:
Definitely. Will update.